packages/react-components/react-tree/src/components/Tree/Tree.types.ts

```diff
@@ -15,7 +15,11 @@ export type TreeSlots = {
 };
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
-export type TreeNavigationData_unstable = { target: HTMLElement; value: TreeItemValue } & (
+export type TreeNavigationData_unstable = {
+  target: HTMLElement;
+  value: TreeItemValue;
+  parentValue: TreeItemValue | undefined;
```

Comment:
hidden breaking change?

Reply:
I believe adding properties here wouldn't be a breaking change?! 🤔 In the sense we're extending the API, not reducing it